src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/UpdateFinancialTransactionDTO.java

```diff
@@ -0,0 +1,15 @@
+package pl.byczazagroda.trackexpensesappbackend.dto;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.PositiveOrZero;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.Instant;
+
+public record UpdateFinancialTransactionDTO(
+        @PositiveOrZero @Digits(integer = 13, fraction = 2) BigDecimal amount,
+        @DateTimeFormat(pattern = "yyyy-mm-dd hh:mm:ss") Instant transactionDate,
+        @Size(max = 255) String description) {
```

Comment:
We need one more field for type of finacial transaction. Remeber about validation Enum type.

Reply:
`FinancialTransactionType` is implemented on other pull-request. The other one is not approved yet.